### ISSUE TITLE
Error code 65 for command xcodebuild after adding the plugin

### ISSUE THREAD

**Issue by andreszs** [chemerisuk/cordova-plugin-firebase-authentication#35]
I've followed the advice from [this related issue](https://github.com/chemerisuk/cordova-plugin-firebase-analytics/issues/39) without success to solve this error:

```
The following build commands failed:
	CompileC /Users/andres/Library/Developer/Xcode/DerivedData/myapp-fzzedyqaehwzihfmlblyztcnfzqu/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/FirebaseCore.build/Objects-normal/x86_64/FIRNetwork.o FirebaseCore/Firebase/Core/FIRNetwork.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/andres/Documents/cordova/myapp/platforms/ios/cordova/build-debug.xcconfig,-workspace,myapp.xcworkspace,-scheme,myapp,-configuration,Debug,-sdk,iphonesimulator,-destination,platform=iOS Simulator,name=iPhone X,build,CONFIGURATION_BUILD_DIR=/Users/andres/Documents/cordova/myapp/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/andres/Documents/cordova/myapp/platforms/ios/build/sharedpch
```

**Steps to reproduce** please tell me if I missed something

- [x] Create blank app (cordova 7.1.0)
- [x] Add platform ios (4.5.4)
- [x] Open `.xcworkspace` in Xcode to:
  - [x] Select my Signing team
  - [x] Add the URL Scheme at the Info tab for my target
  - [x] Enable Push Notifications capability
- [x] Add the plugin
- [x] Add `<resource-file src="GoogleService-Info.plist" />` to config.xml
- [x] Run `pod update` and `pod install` in the `platforms/ios` folder
- [ ] `cordova build ios` will now inevitably fail

The app can be built and deployed from **Xcode** only (making it impossible to use cordova now), and the plugin does work, with these warnings:

```
2018-06-23 20:48:51.901313-0300 myapp[1131:651997] Starting Firebase Authentication plugin
2018-06-23 20:48:51.906258-0300 myapp[1131:652210] 5.3.0 - [Firebase/Core][I-COR000003] The default Firebase app has not yet been configured. Add `[FIRApp configure];` (`FirebaseApp.configure()` in Swift) to your application initialization. Read more: https://goo.gl/ctyzm8.
2018-06-23 20:48:51.918088-0300 myapp[1131:652210] 5.3.0 - [Firebase/Core][I-COR000022] Firebase Analytics is not available
```

How can we fix the `cordova build ios` error 65 now?
Why does the plugin require firebase analytics?
Wasn't the initialization error fixed [here](https://github.com/chemerisuk/cordova-plugin-firebase-authentication/issues/5)?

**Comment by chemerisuk** [comment]
Have you tried to build from Xcode? My cordova build fails from command line as well, but Xcode works without issues (and archives successfully too).

Some firebase libs requires core dependency indeed. Is this error you see prevents app from running?

**Comment by andreszs** [comment]
I can build from **Xcode** only and the app works. But with cordova broken, I can no longer add, edit, or update plugins, I have to uninstall and reinstall your plugin constantly to keep using cordova... could you add the **help needed** tag to this post to see if someone provides a fix? 

A strange thing on the iOS app is this, the **signOut** method calls back the error function instead of the success one:

```javascript
cordova.plugins.firebase.auth.signOut().then(function() {
	// user was signed out
	console.log('signOut: OK'); // Android returns this and shows OK
}, function(error) {
	// error handling
	console.error('signOut: ' + error); // iOS returns this and shows null
});
```

**Comment by andreszs** [comment]
Also, according to [Firebase iOS Guide](https://firebase.google.com/docs/auth/ios/start), you have to add this:
![image](https://user-images.githubusercontent.com/10298071/41827163-610fdae4-7804-11e8-9aed-76b136a846f6.png)

But `AppDelegate.m` in platforms/ios/myapp/Classes/AppDelegate.m has only this:

```
#import "AppDelegate.h"
#import "MainViewController.h"
@implementation AppDelegate
- (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions
{
self.viewController = [[MainViewController alloc] init];
return [super application:application didFinishLaunchingWithOptions:launchOptions];
}
@end
```

Why you are not importing Firebase as the docs say? And the plugin works anyway...

**Comment by chemerisuk** [comment]
Published v1.0.1 that includes fix for method `signOut`
